generators/app/templates/.idea/.idea.StarterKit/.idea/.name

```diff
@@ -0,0 +1 @@
+StarterKit
```

Comment:
Zijn al die ".idea"-folders en files nodig? Er lijkt wel een extra statement hiervoor opgenomen te zijn in de .gitignore-file, maar die files en folder worden precies toch mee opgenomen. Indien niet nodig worden ze best voor toekomstige verwarring te vermijden verwijderd.

Reply:
Nee, goed punt. Ik had het zelf in de gitignore gezet dus het verbaasd mij dat het mee zit. Ik zal het verwijderen.
De .idea folder is overigends de IDE folder van Rider.